<?xml version="1.0" ?>

<launch>
   <include file="$(find gazebo_ros)/launch/empty_world.launch" >
		<arg name="world_name" value="$(find eppacher_hw2)/maps/myworld.xml" />
		<arg name="paused" value="false" />
		<arg name="use_sim_time" value="true" />
		<arg name="gui" value="true" />
	</include>
    <node name="linear_control" pkg="eppacher_hw2" type="linear_control" output="screen"/>

    <param name="robot_description" command="$(find xacro)/xacro --inorder $(find turtlebot3_description)/urdf/turtlebot3_burger_for_autorace.urdf.xacro" />
	<node pkg="gazebo_ros" type="spawn_model" name="spawn_urdf" args="-urdf -model turtlebot3_burger -x 0.0 -y 0.0 -z 0 -Y 0 -param robot_description" />
	<node pkg="rqt_reconfigure" type="rqt_reconfigure" name="rqt_reconfigure" />

</launch>